src/librustdoc/html/render.rs

```diff
@@ -2396,7 +2396,13 @@ fn shorter<'a>(s: Option<&'a str>) -> String {
 #[inline]
 fn plain_summary_line(s: Option<&str>) -> String {
     let line = shorter(s).replace("\n", " ");
-    markdown::plain_summary_line(&line[..])
+    markdown::plain_summary_line_full(&line[..], false)
+}
+
+#[inline]
+fn plain_summary_line_full(s: Option<&str>) -> String {
```

Comment:
I feel like the naming of this is off - this is the version that *truncates* the summary line, not the one that gives the full version. This should be named something like `plain_summary_line_short`.

Also, if you want to make the aliases file also use this version, you can change the call on line 1682 to use this.